### DISCUSSION TITLE
How to get a connected progressbar

### DISCUSSION THREAD

**Discussion by samdotxml** [spectreconsole/spectre.console#421] [Q&A] [answered]
Hello, im currently trying to understand how to get a connected progressbar like yours in your examples.

mine are just `-` but yours are like a full bar

**Comment by phil-scott-78** [comment]
what console are you using? If your console doesn't support unicode (e.g. cmd.exe or the default powershell) you'll get the fallback character display.

**Reply by samdotxml** [reply]
I tested with Windows Terminal and cmd.exe & powershell

**Reply by samdotxml** [reply]
Also while debugging in VS but i guess this is regular cmd.exe . Maybe there is some setting i can set in Windows Terminal or in the Consoleapp of VS

**Comment by patriksvensson** [comment, marked as answer]
It will probably work if you set the terminal's output encoding explicitly:

```powershell
[console]::InputEncoding = [console]::OutputEncoding = New-Object System.Text.UTF8Encoding
```

See this discussion for more information: https://github.com/spectreconsole/spectre.console/discussions/378
